packages/stencil/src/components/datacom-breadcrumb/usage.md

```diff
@@ -3,9 +3,13 @@
 Breadcrumb has one component 'datacom-breadcrumb'
 ### Properties - datacom-breadcrumb
 
--url for passing link of other page. Styled with 'dc- breadcrumb' class.
+-Component  is render separator (>) only when it is wrap with 'div' .
```

Comment:
My previous comment was for ALL the new documentation you added, not just the one line. It's still not too clear what you're trying to convey.